src/framework/parsers/glb-parser.js

```diff
@@ -46,27 +46,14 @@ import { AnimCurve } from '../anim/evaluator/anim-curve.js';
 import { AnimData } from '../anim/evaluator/anim-data.js';
 import { AnimTrack } from '../anim/evaluator/anim-track.js';
 import { Asset } from '../asset/asset.js';
-import { GlbContainerResource } from './glb-container-resource.js';
 import { ABSOLUTE_URL } from '../asset/constants.js';
 
 import { dracoDecode } from './draco-decoder.js';
 
 // resources loaded from GLB file that the parser returns
 class GlbResources {
-    constructor(gltf) {
-        this.gltf = gltf;
-        this.nodes = null;
-        this.scenes = null;
-        this.animations = null;
-        this.textures = null;
-        this.materials = null;
-        this.variants = null;
-        this.meshVariants = null;
-        this.meshDefaultMaterials = null;
-        this.renders = null;
-        this.skins = null;
-        this.lights = null;
-        this.cameras = null;
+    constructor(members) {
```

Comment:
It'd be good to have jsdocs for this constructor, as 'members' does not mean much to me.
But should be try to go strong typed way anyways, where we try to limit dynamic members in the class? Ideally this class would look like TS class with members specified.

Reply:
OK I've create member declarations, though without types for now.